tests/cell/cell_test_suite.cpp

```diff
@@ -462,6 +463,22 @@ class cell_test_suite : public test_suite
         xlnt_assert_equals(cell.number_format().format_string(), "dd--hh--mm");
     }
 
+    void test_number_format_id()
```

Comment:
I would prefer a more meaningful name, like `test_number_format_merge` as the goal is to merge the same formats. Comparing the ids are equal is just a way to check the merge succeeded.